R/report_null.R

```diff
@@ -0,0 +1,71 @@
+#' Creates `NULL` Report
+#'
+#' @param tlg to convert to null report.
+#' @param ind (`integer`) indentation for the outputs of class `VTableTree`.
+#' @param ... not used.
+#'
+#' @rdname report_null
+#' @name report_null
+#'
+#' @returns the `tlg` object or a `NULL` report if the `tlg` is `NULL`, is a `TableTree` with 0 rows, is a `listing_df`
+#'   with 0 rows or is a `list` with 0 elements.
+#'
+#' @export
+#' @examples
+#' report_null(NULL)
+setGeneric("report_null", function(tlg, ...) {
```

Comment:
it seems that the function "std_processing" is renamed to "report_null" (they now have the same functionality). it is more than the issue describes and could you please provide more information about the change? (and for listing table_inset is not applied, is it intentional?)

Reply:
The `std_preprocessing` was just an unexported function wrapping `report_null`. Now that `report_null` can handle multiple types of object, `std_postprocessing` adds complexity without bringing new functionalities. Removing it won't affect user experience and will make the posprocessing function more transparent.
Regarding the listing, unfortunately, I don't think we can apply `table_inset` to a `df_listing` object. That could be a nice feature to add to the package.

Reply:
my concern is more related to the compatibility and the "how the function name reflects the functionality". for first point, it seems fine if users have their call of "report_null". for the second part, I feel like that there is something not really "report_null", and it could possibly contain more processing that is standard in the future?

Reply:
ok, no problem, let me reintroduce `std_postprocessing` so that we can modify it easily in the future.